scripts/rest_examples/reservation-create-aws.http

```diff
@@ -4,11 +4,11 @@ Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
-  "name": "amz-linux-us-east-1",
+  "names": ["amz-linux-us-east-1"],
   "source_id": "1",
   "image_id": "ami-05fa00d4c63e32376",
   "amount": 1,
-  "instance_type": "t1.micro",
+  "instance_type": "t1.micro2",
```

Comment:
This was not meant to be commited, use this to simulate an error. Note that memory jobqueue will panic, so you need to restart the app, then list reservations to see how an error really looks like. In production there is no panic.